docker-compose.yml

```diff
@@ -22,6 +24,8 @@ services:
         build:
             context: .
             dockerfile: Dockerfile.ci
+            args:
+                commit: ${IMAGE_TAG}
```

Comment:
the alternative to the `latest` default is to have a `develop` default but it doesn't help much, only makes it more annoying to use the Dockerfile in isolation (and building should always go through `docker-compose build` to correctly refer to all upstream images)